e2e/utils/index.ts

```diff
@@ -68,7 +68,7 @@ interface RunCmdOpts {
 }
 
 export function currentCli() {
-  return process.env.SELECTED_CLI || 'nx';
+  return 'nx';
 }
```

Comment:
We should probably remove this function altogether and any usage of it should be amended accordingly.